src/CMakeLists.txt

```diff
@@ -38,35 +38,25 @@ macro (upm_target_link_libraries target_name)
 
     # Call the swig link library method for each library
     if (BUILDSWIGNODE)
-      set_target_properties(${SWIG_MODULE_jsupm_${libname}_REAL_NAME} PROPERTIES SKIP_BUILD_RPATH TRUE)
       swig_link_libraries (jsupm_${libname} ${_library})
```

Comment:
For these RPATH changes - you might ask Tudor about them as I think he added them originally.  I believe they were to work around issues with the eclipse IDE(s), but I don't know the specifics.